### PR TITLE
Review service placement howto

### PR DESCRIPTION
**Pull request by pmatulis**

### CODE REVIEW

File: docs/how-to/service-placement.rst [hunk 1]
```diff
@@ -1,71 +1,94 @@
...
-   * - mgr
-     - Ceph Manager Service
-   * - mds
-     - Ceph Metadata Daemon Service
-   * - rgw
-     - Ceph Radosgw Service
+* MON (`Monitor service`_)
+* MDS (`Metadata service`_)
+* MGR (`Manager service`_)
+* RGW (`RADOS Gateway service`_)
```
Comment (sabaini) [inline docs/how-to/service-placement.rst:14]: This sounds like MicroCeph will enable 3x RGW services by default, but that is not the case. MON, MDS and MGRs will be started on the first 3 nodes, but RGW always needs to be enabled by the operator

File: docs/how-to/service-placement.rst [hunk 2]
```diff
@@ -1,71 +1,94 @@
...
+   sudo microceph enable rgw --target node1 --port 8080
+
+Finally, view cluster status again and verify expected changes:
+
+.. code-block:: none
+
+   sudo microceph status
+
+   MicroCeph deployment summary:
+   - node1 (10.111.153.78)
```
Comment (sabaini) [inline docs/how-to/service-placement.rst:82]: It seems strange to me that first `microceph status` reports a node `node1-2c3eb41e-14e8-465d-9877-df36f5d80922` above, but here it's just `node1`?

**Review by sabaini — CHANGES_REQUESTED** [review]
Heya, thanks for this -- two comments inline

File: docs/how-to/service-placement.rst (hunk 1 above)
Comment (pmatulis) [reply docs/how-to/service-placement.rst:14]: You're right. Will adjust.

File: docs/how-to/service-placement.rst (hunk 2 above)
Comment (pmatulis) [reply docs/how-to/service-placement.rst:82]: I was using the existing command output. Maybe we keep it for now and when I get around to actual tool testing I can fill this in with realistic output.

**Comment by pmatulis** [comment]
I made a Git error. I will redo this PR.

**Pull request closed without merging**